calculator.c

```diff
@@ -0,0 +1,147 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+int is_operator(char c)
+{
+    return c == '+' || c == '-' || c == '*' || c == '/';
+}
+
+int operator_precedence(char op)
+{
+    if (op == '*' || op == '/')
+        return 2;
+    if (op == '+' || op == '-')
+        return 1;
+    return 0;

```

Comment:
don't use multiple return statements.

Reply:
I have Resolve this by using conditional statement on new pr (calculator-Review)